kmcos/types.py

```diff
@@ -81,8 +81,12 @@ class Project(object):
         - Processes
     """
 
-    def __init__(self):
+    def __init__(self, **name):
```

Comment:
@meelod 
def __init__(self, **name): 
should be 
def __init__(self, model_name=None):